downloader_app/ci/create_credentials.py

```diff
@@ -8,19 +8,17 @@
 
 load_dotenv()
 
-
 DOWNLADER_PATH = os.path.join(BASE_DIR, 'downloader_app')
-
+CREADENTIALS_PATH = os.path.join(DOWNLADER_PATH, "credentials.json")
```

Comment:
CREDENTIALS_PATH

Reply:
as @fccoelho commented there is still a typo here too